### PR TITLE
I recently upgraded to Arduino 1.6.7 and had to make some changes

### PR DESCRIPTION
**Pull request by firepick1**
I'm using DueTimer in FireStep, which works on Due and Mega2560. The other changes make it happy on Mega2560. I forked DueTimer for stability so I am fine--just thought you might be interested.

(and thanks for DueTimer!)

### CODE REVIEW

**Comment by ivanseidel** [comment]
Hi @firepick1 , Sorry for the delay.. I've been really busy!

What did you implemented? Some sort of way to "bypass" the library if running on other platform?

If so, why didn't you checked that in your code?

(I'm a bit confused on what this really does)

**Comment by firepick1** [comment]
Hey, Ivan,

IDE 1.6.7 was very annoying. I had to add #include <Arduino.h> to all my
own library files as well as my fork of DueTimer.

I have no idea why they made that change. Maybe you'll find a better way to
address that in DueTimer. My problem is solved,
and the pull request is mostly just an FYI in case you haven't bumped into
1.6.7.

BTW, I recognize Due by checking for  _SAM3XA_, which works fine.

I use DueTimer in FireStep, which works on different duinos. Your library
saved me from having to write all that code myself. :D
https://github.com/firepick1/FireStep

Thanks,
Karl

On Sun, Jan 3, 2016 at 2:49 PM, Ivan Seidel <notifications@github.com>
wrote:

> Hi @firepick1 <https://github.com/firepick1> , Sorry for the delay.. I've
> been really busy!
>
> What did you implemented? Some sort of way to "bypass" the library if
> running on other platform?
>
> If so, why didn't you checked that in your code?
>
> (I'm a bit confused on what this really does)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ivanseidel/DueTimer/pull/44#issuecomment-168549330>.
>



-- 
Karl Lew
FIREPICK SERVICES LLC
karl@firepick.org
www.firepick.org

File: DueTimer.h [hunk 1]
```diff
@@ -102,8 +102,8 @@ extern DueTimer Timer6;
 extern DueTimer Timer7;
 extern DueTimer Timer8;
 
+// _SAM3XA_
 #endif
 
-#else
-	#error Oops! Trying to include DueTimer on another device?
```
Comment (ivanseidel) [inline DueTimer.h]: Let's keep that Error! It saves people sometimes when forgetting to change the board...

**Comment by ivanseidel** [comment]
Got it now! Just change it to keep the error, and I will merge.

I have to bump version after that, but don't mind that.. I will do it here after merging

**Comment by firepick1** [comment]
OK. All set!

I released a new FireStep to match up with the reverting of DueTimer.h

**Comment by ivanseidel** [comment]
Done ;)

**Pull request merged**